### ISSUE TITLE
Add issue templates available across the epiverse-trace organisation

### ISSUE THREAD

**Issue by Bisaloo** [epiverse-trace/.github#1]
linelist provides generic issue templates we might want to re-use. They are already re-used in epiparameters.

https://github.com/epiverse-trace/linelist/tree/main/.github/ISSUE_TEMPLATE

https://github.com/epiverse-trace/linelist/tree/main/.github/PULL_REQUEST_TEMPLATE

**Comment by avallecam** [comment]
I tested this and couldn't replicate the expected behaviours of the [default health community files](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) for issue templates. 

I replicated the scenario in my personal repo and, as far as I looked, issue templates are not propagated in the same way as the CoC and Contrib files.

The current approach used in `packagetemplate` seems to be the one that works, letting it be the propagator of issue and PR templates for each new package, customizable from the settings tab... until someone proves otherwise n.n

Still an issue for older or external packages.
